docs/builtins.md

```diff
@@ -297,33 +342,86 @@ Returns nothing (`*`).
 ## Numeric operations
 
 ### log
+
 ```py
 def log(x: f24, base: f24) -> f24
 ```
+
 Computes the logarithm of `x` with the specified `base`.
 
 ### atan2
+
 ```py
 def atan2(x: f24, y: f24) -> f24
 ```
+
 Computes the arctangent of `y / x`.
 
 Has the same behaviour as `atan2f` in the C math lib.
 
 ### to_f24
+
 ```py
 def to_f24(x: any number) -> f24
 ```
+
 Casts any native number to an f24.
 
 ### to_u24
+
 ```py
 def to_u24(x: any number) -> u24
 ```
+
 Casts any native number to a u24.
 
 ### to_i24
+
 ```py
 def to_i24(x: any number) -> i24
 ```
+
 Casts any native number to an i24.
+
+## Encoding functions
+
+### Bytes/decode_utf8
+
+```python
+def Bytes/decode_utf8(bytes: [u24], out: String) -> String
+```
+
+Decodes a sequence of bytes to a String using utf-8 encoding.
+
+### Bytes/decode_ascii
+
+```python
+def Bytes/decode_ascii(bytes: [u24]) -> String
+```
+
+Decodes a sequence of bytes to a String using ascii encoding.
+
+### String/encode_utf8
+
+```python
+def String/encode_utf8(s: String, out: [u24]) -> [u24]
+```
+
+Encodes a String to a sequence of bytes using utf-8 encoding.
+
+### String/encode_ascii
+
+```python
+def String/encode_ascii(s: String) -> [u24]
+```
+
+Encodes a String to a sequence of bytes using ascii encoding.
+
+### Utf8/decode_rune
```

Comment:
Do we want to commit to calling unicode codepoints "runes"?
Not saying we shouldn't, but i'd like to hear your reasoning.